### ISSUE TITLE
Any other notification type?

### ISSUE THREAD

**Issue by tak3joy** [Aioros/destiny-reminders#7]
I want just to suggest use browser notification as other option if it possible, it will be received on any device, and will not have so much troubles like emails. Sorry for my English.

**Comment by Aioros** [comment]
I'm not sure, to be honest. It's an interesting option, but what happens if I create a reminder on Chrome on my home desktop, and then I'm

**Comment by Aioros** [comment]
Thanks, it's an interesting idea and I'm currently investigating it.

The thing that I like about emails is that they are completely platform-agnostic. You can set a reminder on any of your devices and get the email on any other one. I think push notification are a little more restrictive: if I subscribe on my work PC I won't get notifications when I'm at home, or on my phone.

On the other hand, I like the idea of not needing any user info at all. I'll consider adding it as an option.

**Comment by Diegopie** [comment]
Have you considered building out a service worker so the app can use Push API notifications? I'm fairly new to this stuff and it was only briefly mentioned during school but it's something that would be neat for this application and something I'd like to practice for my own apps

**Comment by Aioros** [comment]
Yes, I suppose that's the plan. I'm currently working on another issue first, then I intend to start with this.

**Comment by Diegopie** [comment]
Looking forward to it!

**Comment by ChiefGrimm** [comment]
I just created a GitHub account just to concur with tak3joy. Another notification type would be an awesome addition
